hosted_control_planes/hosted-control-planes-release-notes.adoc

```diff
@@ -87,6 +87,8 @@ With this release, you can configure the proxy on the Konnectivity sidecar of th
 
 * Previously, the bulk destroy option in the {mce} console for {VirtProductName} did not destroy a hosted cluster. With this release, this issue is resolved. (link:https://issues.redhat.com/browse/ACM-10165[*ACM-10165*])
 
+* Previously, updating the `additionalTrustBundle` parameter in a {hcp} cluster configuration did not get applied to compute nodes. With this release, a fix ensures that updates to the `additionalTrustBundle` parameter automatically apply to compute nodes that exist in your {hcp} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-36680[*OCPBUGS-36680*])
```

Comment:
Updating to the version containing this fix would result in an automatic rollout of the existing Nodes to pick the bundle. This needs to be clearly state to adjust consumers expectations

Reply:
Thanks for your review, @enxebre. I updated the text. When you can, let me know what you think.